server/routerlicious/packages/services/src/mongodb.ts

```diff
@@ -445,6 +445,36 @@ export class MongoDb implements core.IDb {
 }
 
 export type ConnectionNotAvailableMode = "ruleBehavior" | "stop"; // Ideally we should have 'delayRetry' options, but that requires more refactor on our retry engine so hold for this mode;
+const DefaultMongoDbMonitoringEvents = [
+	"serverOpening",
+	"serverClosed",
+	"serverDescriptionChanged",
+	"topologyOpening",
+	"topologyClosed",
+	"topologyDescriptionChanged",
+	"serverHeartbeatStarted",
+	"serverHeartbeatSucceeded",
+	"serverHeartbeatFailed",
+	// "commandStarted", Comment out because this will be too often
+	// "commandSucceeded", Comment out because this will be too often
+	// "commandFailed", Comment out because this will be too often
```

Comment:
I am curious as to how you figured out which of these events will be frequent and which will not..I thought it might be safer to add the Monitoring event list to the config so that we can turn it off and on if needed..but I don't have a strong opinion on this.

Reply:
I got the list and their meaning by reading: https://www.mongodb.com/docs/drivers/node/v4.13/fundamentals/monitoring/cluster-monitoring/

And further, this list is overrideable by config from:
`this.dbMonitoringEventsList = dbMonitoringEventsList ?? DefaultMongoDbMonitoringEvents;`